options/locale/locale_en-US.ini

```diff
@@ -1755,7 +1755,7 @@ milestones.no_due_date = No due date
 milestones.open = Open
 milestones.close = Close
 milestones.new_subheader = Milestones organize issues and track progress.
-milestones.completeness = %d%% Completed
+milestones.completeness = <strong>%d%%</strong> Completed
```

Comment:
I would have really liked to use `{percentage} Completed` here where the template code takes care of percentage rendering, but it's currently not possible with this translation system to have such tokens.

Reply:
I could put `%s completed` and avoid the HTML in the translation, but that might confuse translators because they don't know that `%s` is supposed to be a percentage.

Reply:
Is it not preferred, do don't change existing translations?

Reply:
Well… Yesn't.
The problem is simply that new strings (which is the case for modified strings) can be untranslated for quite some time.